org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/changes/TestEditorChangeDetector.xtend

```diff
@@ -13,12 +14,24 @@ class TestEditorChangeDetector implements ChangeDetector {
 	@Inject GradleBuildChangeDetector gradleChangeDetector
 	@Inject ChangeFilter filter
 
-	var injectorChain = memoize[ #[gitChangeDetector, gradleChangeDetector, filter] ]
+	var injectorChain = memoize[#[gitChangeDetector, gradleChangeDetector, filter]]
```

Comment:
I know I introduced this variable, but I think its naming was kind of a Freudian slip: I'm pretty sure I meant to call it _detectorChain_, because that seems to make a lot more sense…